### PR TITLE
Highlighted text for search

### PR DESCRIPTION
**Pull request by akrol95**

### CODE REVIEW

File: BMM.Core/BMM.Core.csproj [hunk 1]
```diff
@@ -47,6 +47,10 @@
       </Reference>
     </ItemGroup>
 
+    <ItemGroup>
+      <Folder Include="Implementations\Share\" />
```
Comment (kkuepper) [inline BMM.Core/BMM.Core.csproj:51]: probably just delete the folder

File: BMM.Core/Translation/en/main.json [hunk 2]
```diff
@@ -515,5 +515,14 @@
     "PlaysSingular": "{0} play",
     "AddToMyPlaylist": "Add playlist to favorites",
     "SuccessfullyAddedToPlaylists": "Successfully added to favourites"
+  },
+  "HighlightedTextTrackViewModel": {
+    "AutoTranscribed": "Auto-transcribed",
+    "AddTo": "Add to...",
```
Comment (kkuepper) [inline BMM.Core/Translation/en/main.json:521]: It's supposed to be "Add to playlist" and only show ... when there's not enough space.

File: BMM.UI.iOS/Resources/Images.xcassets/IconAI.imageset/icon_ai.pdf
Comment (kkuepper) [inline BMM.UI.iOS/Resources/Images.xcassets/IconAI.imageset/icon_ai.pdf:1]: pdf?
Is that correct?

Comment (akrol95) [reply BMM.UI.iOS/Resources/Images.xcassets/IconAI.imageset/icon_ai.pdf:1]: Yep, that's the native format for vector graphic for iOS.

File: BMM.Core/Translation/en/main.json (hunk 2 above)
Comment (akrol95) [reply BMM.Core/Translation/en/main.json:521]: done

**Pull request merged**